src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -350,6 +354,41 @@ export class StudyViewPageStore {
             )
         );
 
+        this.reactionDisposers.push(
```

Comment:
i think i understand why you made this a reaction (we only want to force the tooltip on when it changes, not after user closes it)?

Reply:
yes

Reply:
I'm confused though - why would the shared groups change after initial page load? i.e. why a reaction, why not just in constructor?

Reply:
true. I'll remove it